assets/src/scripts/components/hydrograph/domain.js

```diff
@@ -101,8 +105,23 @@ export const yDomainSelector = createSelector(
  * @param {Object} yScale - d3 scale
  * @returns {Array} of tick values
  */
-export const getYTickDetails = function (yDomain) {
-    const tickValues = ticks(yDomain[0], yDomain[1], Y_TICK_COUNT);
+export const getYTickDetails = function (yDomain, parmCd) {
+    const isSymlog = SYMLOG_PARMS.indexOf(parmCd) > -1;
+    let tickCount = Y_TICK_COUNT;
+    if (isSymlog) {
+        tickCount = Y_TICK_COUNT_SMALL_LOG;
+        if (mediaQuery(config.USWDS_SMALL_SCREEN)) {
+            tickCount++;
+        }
+        if (mediaQuery(config.USWDS_MEDIUM_SCREEN)) {
+            tickCount++;
+        }
+        if (mediaQuery(config.USWDS_LARGE_SCREEN)) {
+            tickCount++;
+        }
+    }
+
+    const tickValues = ticks(yDomain[0], yDomain[1], tickCount);
```

Comment:
This ticks function treats the tickCount as a "suggestion", so it's sometimes unpredictable. This method of choosing the count is working for the test sites I've encountered, but does often lead towards only a top and bottom tick on small widths.